src/c_kzg_4844.c

```diff
@@ -990,14 +990,25 @@ C_KZG_RET blob_to_kzg_commitment(
     return C_KZG_OK;
 }
 
+/**
+ * Struct holding the argument to verify_kzg_proof_impl.
+ * Having a struct holding these allows some code simpliciations
+ *
+ * An instance of this struct models a claim (to be verified) that:
+ * proof_g1 is a proof for p(evaluation_point) == y_fr,
+ * where commitment_g1 is a commitment to the polynomial p.
+ */
+typedef struct {
```

Comment:
It's better to provide the name of the struct here. Please look at https://stackoverflow.com/questions/1675351/typedef-struct-vs-struct-definitions